doc/sphinx/language/core/modules.rst

```diff
@@ -198,7 +198,7 @@ are now available through the dot notation.
    If :n:`@module_binder`\s are specified, declares a functor with parameters given by the list of
    :token:`module_binder`\s.
 
-.. cmd:: Import {+ @filtered_import }
+.. cmd:: Import {? @import_categories } {+ @filtered_import }
 
    .. insertprodn filtered_import filtered_import
```

Comment:
```suggestion
   .. insertprodn import_categories filtered_import
```

This is to include the definition of `import_categories` in the manual.

Please add a sentence here describing what `import_categories` does.